src/useAsObservableSource.ts

```diff
@@ -0,0 +1,15 @@
+import { useDeprecated } from "./utils"
+import { observable, runInAction } from "mobx"
+import { useState } from "react"
+
+export function useAsObservableSource<TSource>(current: TSource): TSource {
+    if ("production" !== process.env.NODE_ENV)
+        useDeprecated(
+            "[mobx-react-lite] 'useAsObservableSource' is deprecated, please store the values directly in an observable, for example by using 'useObservable', and sync with 'useEffect' when needed"
```

Comment:
I suppose that's a typo? I hope you are no planning on returning `useObservable` again :)
```suggestion
            "[mobx-react-lite] 'useAsObservableSource' is deprecated, please store the values directly in an observable, for example by using 'useLocalStore', and sync with 'useEffect' when needed"
```

Anyway, I think this deprecation should probably have a link to a documentation site with a more thorough explanation.

Reply:
Yes, didn't get to that yet